<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>953</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gTdsParams">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>521</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>TDS Pulse Parameters for polarisation sweep:</string>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>501</width>
       <height>171</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lblTdsStart">
           <property name="text">
            <string>start (ps):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lEditTdsStart"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="lblTdsEnd">
           <property name="text">
            <string>end (ps):  </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lEditTdsEnd"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lblTdsAvgs">
           <property name="text">
            <string>avgs:        </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lEditTdsAvgs"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="lblAngle1">
           <property name="text">
            <string>angle1:        </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lEditAngle1"/>
         </item>
         <item>
          <widget class="QLabel" name="lblAngle2">
           <property name="text">
            <string>angle2:        </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lEditAngle2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="lblTdsFrames">
           <property name="text">
            <string>frames</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lEditFrames"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="btnStart">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStop">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxCreateGIF">
         <property name="text">
          <string>Create GIF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_keepSrcImgs">
         <property name="text">
          <string>keep source images</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>THEA PHI SCAN</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStatus">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>880</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ScanControl Status: Idle</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Overall progress:</string>
    </property>
    <widget class="QProgressBar" name="progPolSweep">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="lblFrameCount">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Current angle: </string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gLivePlot">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>521</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string>THz Pulse Data</string>
    </property>
    <widget class="PlotWidget" name="livePlot" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>491</width>
       <height>391</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="xyLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>cursor position: (- , -)</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnAnimateResult">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>890</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Animate result</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>310</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Scan Progress</string>
    </property>
    <widget class="QProgressBar" name="progAvg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="lblMaterial">
       <property name="text">
        <string>Material:        </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lEditMaterial"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
